colossalai/communication/p2p.py

```diff
@@ -136,12 +136,12 @@ def _communicate(tensor_send_next=None,
 def recv_forward(input_tensor_shape, prev_rank=None, dtype=torch.float, scatter_gather_tensors=False):
     """Receives the input tensor from the previous member in pipeline.
 
-    :param input_tensor_shape: The shape of the tensor to be recieved
-    :param prev_rank: The rank of the source of the tensor
-    :type input_tensor_shape: torch.Size
-    :type prev_rank: int, optional
-    :return: The input tensor in forward step
-    :rtype: :class:`torch.Tensor`
+    Args:
+        input_tensor_shape (:class:`torch.Size`): The shape of the tensor to be received.
+        prev_rank (int, optional): The rank of the source of the tensor.
+
+    Returns:
+        :class:`torch.Tensor`: The input tensor in forward step.
```

Comment:
Very confusing. The returned tensor is actually the output of the forward step of the previous pipeline stage